### ISSUE TITLE
Is v2 coming out?

### ISSUE THREAD

**Issue by Ikenw** [niyankhadka/crypto-wallet-bruteforce#7]
First of all thank you very much for creating this tool. I read in a different discussion that you were planning on releasing v2 version of this tool and I wanted to ask if you’re still planning to do so.

**Comment by niyankhadka** [comment]
Yes i am planning. Been so busy that i am not able to update the repository. I will soon update this repository. By end of this July, i will update all the codes.

**Comment by Kilanyapp** [comment]
We still waiting it

**Comment by niyankhadka** [comment]
Yes, just launched it. 

Please do check latest version 2.0.0 👍
